packages/ui/src/components/NumberInput/NumberInput.stories.tsx

```diff
@@ -45,8 +45,8 @@ export const Disabled = () => (
 
 export const Pseudoclasses = () => (
   <HStack>
-    <NumberInput size="md" inputMode="decimal" disabled={false} />
-    <NumberInput size="md" inputMode="numeric" disabled={false} value={1} max={0} />
-    <NumberInput size="md" inputMode="numeric" disabled={false} isFocus />
+    <NumberInput size="md" />
```

Comment:
💭 I think, there is no need to define size when you use the default value for it. Also as you named this story `Pseudoclasses` you should change only props related to them, not the size.

Reply:
But in this component size property is required so i need to define size there

Reply:
So there is the catch - size shouldn't be a required property